sdk/appconfiguration/ci.yml

```diff
@@ -22,6 +22,7 @@ trigger:
     include:
     - sdk/appconfiguration/
     - sdk/core/
+    - tools/
```

Comment:
Was `eng/` deliberately excluded here before? `tools/` is what is being asked for, but we should also be doing it for `eng/` correct?

Reply:
I had missed it earlier. Thanks for the comment. I have fixed all yml file now.

Reply:
I'd suggest not triggering all based on eng changes.